Makefile

```diff
@@ -83,8 +83,8 @@ production:
 	@if [ -f data.yml ]; then rm data.yml; fi
 	@echo "---" >> data.yml
 	@echo "CF_SPACE: production" >> data.yml
-	@echo "MIN_INSTANCE_COUNT_TEMPLATE_PREVIEW: 8" >> data.yml
-	@echo "MAX_INSTANCE_COUNT_TEMPLATE_PREVIEW: 40" >> data.yml
+	@echo "MIN_INSTANCE_COUNT_TEMPLATE_PREVIEW: 1" >> data.yml
```

Comment:
Also if we wish to have the minimum overnight be 2 then should we allow this to be 1 ever? Should we instead set the minimum at 2? Also given we have app crashes occasionally I would be hesitant about ever allowing any of our sync apps to be allowed to only just run a single instance?